assembly/src/assembler/tests.rs

```diff
@@ -391,3 +392,44 @@ fn module_ordering_can_be_arbitrary() -> Result<(), Report> {
 
     Ok(())
 }
+
+#[test]
+fn can_assemble_a_multi_module_kernel() -> Result<(), Report> {
+    const KERNEL: &str = r#"
+        use.kernellib::helpers->h
+        export.foo
+            exec.h::get_caller
+        end"#;
+    const HELPERS: &str = r#"
+        export.get_caller
+            caller
+        end"#;
+    const PROGRAM: &str = r#"
+        begin
+            syscall.foo
+        end"#;
+
+    let context = TestContext::new();
+
+    let kernel_lib = {
+        let helpers = context.parse_module_with_path(
+            LibraryPath::new_from_components(
+                LibraryNamespace::User("kernellib".into()),
+                [Ident::new("helpers").unwrap()],
+            ),
+            HELPERS,
+        )?;
+        let kernel = context.parse_kernel(KERNEL).unwrap();
+        std::println!("{kernel}");
```

Comment:
This print statement can be removed, right?

Reply:
Ah yep, was just using it to verify some things about the AST